BoostRunClub/SceneDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SceneDelegate.swift
+//  BoostRunClub
+//
+//  Created by 김신우 on 2020/11/19.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+    var window: UIWindow?
+    var appCoordinator: AppCoordinator?
```

Comment:
```suggestion
    private var appCoordinator: AppCoordinator?
```
사소한 부분이지만 접근제어를 가능한 닫아두는 것이 유지보수에 좋습니다.
예를들어 외부에서 appCoordinator에 접근해서 변경한다면 일관성 있는 동작을 보장하기 어려워집니다.